### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Adesoji1** [jfilter/split-folders#46]
I would want splitfolders to have a package that can split csv files having labels of fake and real in them and be splitted into train_fake.csv and test_fake.csv files.  and Train_real.csv and Test_real.csv files respectively

**Comment by jfilter** [comment]
Thanks for the suggestion\! This is out of scope for split-folders, which operates on folder/file structures on disk. Splitting CSV files by label is better handled with pandas or similar data-wrangling tools — e.g. `df.groupby('label')` followed by `train_test_split` from scikit-learn.
